packages/dev/docs/src/docs.js

```diff
@@ -46,6 +51,44 @@ for (let link of links) {
   link.parentNode.replaceChild(container, link);
 }
 
+function ImageModal({children}) {
+  let [trigger, contents] = React.Children.toArray(children);
+  let [isOpen, setOpen] = React.useState(false);
+  let {pressProps} = usePress({
+    onPress: () => setOpen(true)
+  });
+
+  trigger = React.cloneElement(trigger, pressProps);
+  return (
+    <>
+      {trigger}
+      <Modal isDismissable isOpen={isOpen} onClose={() => setOpen(false)} UNSAFE_style={{overflow: 'scroll'}}>
+        <FocusScope contain restoreFocus autoFocus>
+          {contents}
+        </FocusScope>
+      </Modal>
+    </>
+  );
+}
+
+for (let image of images) {
+  let container = document.createElement('span');
+  let url = image.src.replace(/.*\/\/[^/]*/, '');
+
+  ReactDOM.render(
+    <ThemeProvider UNSAFE_className={docsStyle.inlineProvider}>
+      <ImageModal>
+        {/* eslint-disable-next-line jsx-a11y/no-noninteractive-element-to-interactive-role */}
+        <img role="button" tabIndex={0} src={url} className={image.className} alt={image.alt} />
+        {/* eslint-disable-next-line jsx-a11y/no-noninteractive-element-to-interactive-role */}
+        <img role="button" tabIndex={0} src={url} alt={image.alt} />
```

Comment:
alternative might be to make these `<input type="image" />` but didn't feel as appropriate cuz those are mean for form submit buttons?

Reply:
So had no idea about that, I've now done some reading about it
It does appear that input type=image is definitely meant as a submit
Instead, I propose
<button><img /></button>
that should get rid of the need for all these workarounds like the tab index and ignore warnings

Reply:
think you forgot to paste in a code block above :P

Reply:
not forgotten, just not escaped ;P